### DISCUSSION TITLE
it is too slow to chat with the chatbot

### DISCUSSION THREAD

**Discussion by jensenw-pki** [arc53/DocsGPT#781] [General]
I deployed the DocsGPT in AWS environment. But it is too slow to chat with it, it needs several minutes to response an answer for each question.
what is the potential problem? it seems work perfectly in the official website https://docsgpt.arc53.com/

**Comment by dartpain** [comment]
How did you deploy it? 
Are you using sagemaker? 
Streaming responses shoud be just as fast as anywhere else.

Unless you are using llama.cpp

**Reply by jensenw-pki** [reply]
I just create an EC2 instance.

**Reply by dartpain** [reply]
ok, so you used the ec2 guide here: https://docs.docsgpt.co.uk/Deploying/Hosting-the-app
right?

And for inference you are using OpenAI?
